staging/src/k8s.io/kubectl/pkg/cmd/apply/apply.go

```diff
@@ -520,22 +535,25 @@ See http://k8s.io/docs/reference/using-api/api-concepts/#conflicts`, err)
 
 			patcher, err := newPatcher(o, info)
 			if err != nil {
-				return err
+				errs = append(errs, err)
+				continue
 			}
 			patchBytes, patchedObject, err := patcher.Patch(info.Object, modified, info.Source, info.Namespace, info.Name, o.ErrOut)
 			if err != nil {
-				return cmdutil.AddSourceToErr(fmt.Sprintf("applying patch:\n%s\nto:\n%v\nfor:", patchBytes, info), info.Source, err)
+				errs = append(errs, cmdutil.AddSourceToErr(fmt.Sprintf("applying patch:\n%s\nto:\n%v\nfor:", patchBytes, info), info.Source, err))
+				continue
 			}
 
 			info.Refresh(patchedObject, true)
 
 			if string(patchBytes) == "{}" && !o.shouldPrintObject() {
 				printer, err := o.ToPrinter("unchanged")
 				if err != nil {
-					return err
+					errs = append(errs, err)
+					continue
 				}
 				if err = printer.PrintObj(info.Object, o.Out); err != nil {
-					return err
+					errs = append(errs, err)
```

Comment:
Nit: I would leave continue here, in case someone adds something after the condition.